shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/module/HomeViewModel.kt

```diff
@@ -30,16 +35,40 @@ class HomeViewModel(
     private var currentPage = 1
 
     fun fetchData() {
-        getSurveysUseCase(currentPage, 10, isForceLatestData = false)
+        getProfile()
```

Comment:
We should do it in the Integrate task? 🤔

Reply:
Nope, this is for testing purposes